mmv1/products/compute/api.yaml

```diff
@@ -7347,7 +7346,7 @@ objects:
           - :BPS_10G
           - :BPS_20G
           - :BPS_50G
-      - !ruby/object:Api::Type::String
+      - !ruby/object:Api::Type::Enum
```

Comment:
Let's keep this typed as a string, we don't get much from making it an enum.

Reply:
Can do